examples/insurance/src/main/ml-modules/root/custom-modules/custom/enrichAdvantage/main.sjs

```diff
@@ -0,0 +1,114 @@
+/*
+  Copyright 2012-2019 MarkLogic Corporation
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+const DataHub = require("/data-hub/5/datahub.sjs");
+const datahub = new DataHub();
+// Require module for mapping latitude and longitude from zip codes
+const zipcodeData = require("/custom-modules/utils/zipcodeData.sjs");
```

Comment:
This file is fairly large for adding to version control - I recommend slimming it down to a handful of zip codes that will guarantee some matches against our ingested data.

Reply:
Yes, there are quite a few postal code not covered and we get null. I will reduce it to around 100 or so to match ingested data.